{
  "Faults": [
    {
      "latitude": 51.0734495606056,
      "type": "PSI",
      "reference": "218172",
      "etaUpdate": true,
      "affectedCustomerCount": 73,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "1629352b-854e-407c-b2ce-5c18b9234430",
      "minLongitude": -1.3998,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3689807216062622,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-08-13T07:30:00.000+0000",
      "estimatedRestoration": "2023-08-19T15:30:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '218172'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/218172",
      "maxLongitude": -1.3481,
      "minLatitude": 51.0568,
      "name": "SO21 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO21 2FH",
        "SO21 2LP",
        "SO21 2LR",
        "SO21 2NR",
        "SO21 2NZ",
        "SO21 2PA",
        "SO22 5QL",
        "SO22 5QS",
        "SO22 5RA",
        "SO22 5RB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3998,
              51.0848
            ],
            [
              -1.3481,
              51.0848
            ],
            [
              -1.3481,
              51.0568
            ],
            [
              -1.3998,
              51.0568
            ],
            [
              -1.3998,
              51.0848
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.0848,
      "etrUpdate": true
    },
    {
      "latitude": 51.24194247061283,
      "type": "HV",
      "reference": "MH4069",
      "etaUpdate": true,
      "affectedCustomerCount": 3,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "1831bc3b-6b5f-4716-99a4-6afd6118afec",
      "minLongitude": -1.3054,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.291741728516251,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-08-14T13:29:00.000+0000",
      "estimatedRestoration": "2023-08-15T11:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MH4069'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MH4069",
      "maxLongitude": -1.2748,
      "minLatitude": 51.2348,
      "name": "RG25 & RG28 Areas",
      "otherUpdate": true,
      "affectedAreas": [
        "RG25 3DQ",
        "RG28 7NT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3054,
              51.2538
            ],
            [
              -1.2748,
              51.2538
            ],
            [
              -1.2748,
              51.2348
            ],
            [
              -1.3054,
              51.2348
            ],
            [
              -1.3054,
              51.2538
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-08-14T16:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2538,
      "etrUpdate": true
    },
    {
      "latitude": 51.71621337140821,
      "type": "LV",
      "reference": "MH4399",
      "etaUpdate": true,
      "affectedCustomerCount": 12,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "55cd796b-f173-4a58-91a5-de6c591fe0ee",
      "minLongitude": -1.9666,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.9632737005187648,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-08-14T22:46:00.000+0000",
      "estimatedRestoration": "2023-08-15T10:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MH4399'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MH4399",
      "maxLongitude": -1.9613,
      "minLatitude": 51.7155,
      "name": "GL7 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GL7 1ED",
        "GL7 2PF",
        "GL7 2PP",
        "GL7 2PU",
        "GL7 2PW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.9666,
              51.7172
            ],
            [
              -1.9613,
              51.7172
            ],
            [
              -1.9613,
              51.7155
            ],
            [
              -1.9666,
              51.7155
            ],
            [
              -1.9666,
              51.7172
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-08-15T00:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.7172,
      "etrUpdate": true
    },
    {
      "latitude": 51.42494876745764,
      "type": "PSI",
      "reference": "218023",
      "etaUpdate": true,
      "affectedCustomerCount": 46,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "aa35ae2a-92de-45c3-b82e-6fd4bf8de137",
      "minLongitude": -1.4213,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.4057196979697988,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-08-14T23:00:00.000+0000",
      "estimatedRestoration": "2023-08-15T15:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '218023'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/218023",
      "maxLongitude": -1.3903,
      "minLatitude": 51.4143,
      "name": "RG20 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG20 8EU",
        "RG20 8PD",
        "RG20 8PE",
        "RG20 8PG",
        "RG20 8PH",
        "RG20 8PT",
        "RG20 8PU",
        "RG20 8PX"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.4213,
              51.4317
            ],
            [
              -1.3903,
              51.4317
            ],
            [
              -1.3903,
              51.4143
            ],
            [
              -1.4213,
              51.4143
            ],
            [
              -1.4213,
              51.4317
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4317,
      "etrUpdate": true
    },
    {
      "latitude": 50.66061911901488,
      "type": "LV",
      "reference": "MH4407",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-08-15T01:20:00.000+0000",
      "affectedCustomerCount": 18,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "80f118a6-eae5-4239-9835-1b743e9be167",
      "minLongitude": -1.4137,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3789696649655938,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-08-14T23:35:00.000+0000",
      "estimatedRestoration": "2023-08-15T09:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MH4407'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MH4407",
      "maxLongitude": -1.2985,
      "minLatitude": 50.6396,
      "name": "PO30 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO30 4BT",
        "PO30 4BX",
        "PO30 4DH",
        "PO30 4DL",
        "PO30 4DR",
        "PO30 4DU",
        "PO30 5BE",
        "PO30 5HS"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.4137,
              50.7073
            ],
            [
              -1.2985,
              50.7073
            ],
            [
              -1.2985,
              50.6396
            ],
            [
              -1.4137,
              50.6396
            ],
            [
              -1.4137,
              50.7073
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-08-15T01:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7073,
      "etrUpdate": true
    },
    {
      "latitude": 51.000287754435895,
      "type": "LV",
      "reference": "MH4394",
      "etaUpdate": true,
      "affectedCustomerCount": 10,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "b4bee38b-a771-4602-894d-522f3075a396",
      "minLongitude": -0.6959,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.6813075482663363,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-08-14T22:28:00.000+0000",
      "estimatedRestoration": "2023-08-15T10:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MH4394'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MH4394",
      "maxLongitude": -0.6742,
      "minLatitude": 50.9945,
      "name": "GU28 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GU28 9BL",
        "GU28 9BN",
        "GU28 9BW",
        "GU28 9BY",
        "GU28 9BZ",
        "GU28 9DQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.6959,
              51.0033
            ],
            [
              -0.6742,
              51.0033
            ],
            [
              -0.6742,
              50.9945
            ],
            [
              -0.6959,
              50.9945
            ],
            [
              -0.6959,
              51.0033
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.0033,
      "etrUpdate": true
    },
    {
      "latitude": 50.99791526431056,
      "type": "LV",
      "reference": "MH4425",
      "etaUpdate": true,
      "affectedCustomerCount": 14,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "fd975e20-7cdb-4b96-a7a7-93a89b84008c",
      "minLongitude": -1.389,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3826311313208373,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-08-15T02:32:00.000+0000",
      "estimatedRestoration": "2023-08-15T07:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MH4425'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MH4425",
      "maxLongitude": -1.378,
      "minLatitude": 50.9962,
      "name": "SO53 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO53 5NP",
        "SO53 5NQ",
        "SO53 5NR",
        "SO53 5NS",
        "SO53 5NT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.389,
              50.9992
            ],
            [
              -1.378,
              50.9992
            ],
            [
              -1.378,
              50.9962
            ],
            [
              -1.389,
              50.9962
            ],
            [
              -1.389,
              50.9992
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-08-15T05:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9992,
      "etrUpdate": true
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-08-15T05:01:00.000+0000",
      "latitude": 51.59161244644287,
      "estimatedRestoration": "2023-08-15T08:05:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MH4433'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MH4433",
      "reference": "MH4433",
      "etaUpdate": true,
      "affectedCustomerCount": 29,
      "name": "SN3 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN3 4UE",
        "SN3 4UG",
        "SN3 4UJ",
        "SN3 4UR",
        "SN3 4UT",
        "SN3 4UU"
      ],
      "location": {
        "coordinates": [
          -1.7538187891774768,
          51.59161244644287
        ],
        "type": "Point"
      },
      "UUID": "3dadc08b-b704-4eee-8c88-cb468bedf94c",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -1.7538187891774768,
      "resolved": false
    },
    {
      "latitude": 57.3476692140057,
      "type": "LV",
      "reference": "MH4432",
      "etaUpdate": true,
      "affectedCustomerCount": 2,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "a5299401-cd53-4546-ab88-f121b40d7460",
      "minLongitude": -2.6764,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.6563490540398034,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-08-15T04:44:00.000+0000",
      "estimatedRestoration": "2023-08-15T08:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MH4432'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MH4432",
      "maxLongitude": -2.6362,
      "minLatitude": 57.3395,
      "name": "AB52 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "AB52 6LN",
        "AB52 6LP"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.6764,
              57.3554
            ],
            [
              -2.6362,
              57.3554
            ],
            [
              -2.6362,
              57.3395
            ],
            [
              -2.6764,
              57.3395
            ],
            [
              -2.6764,
              57.3554
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.3554,
      "etrUpdate": true
    }
  ]
}
